src/app/page.js

```diff
@@ -0,0 +1,420 @@
+"use client";
+
+import "./app.css";
+import "@appwrite.io/pink";
+import "@appwrite.io/pink-icons";
+import { useState, useEffect, useRef } from "react";
```

Comment:
```suggestion
import { useState, useEffect, useRef, useCallback } from "react";
```

`useCallback` is being used later but has not been defined